src/classes/RD2_RecurringDonationsOpp_TDTM.cls

```diff
@@ -94,9 +94,15 @@ public class RD2_RecurringDonationsOpp_TDTM extends TDTM_Runnable {
             return;
         }
 
-        new RD2_OpportunityEvaluationService()
-            .withRds(rdIds)
-            .handleOutdatedRecurringDonations();
+        RD2_QueueableService.EvaluateInstallmentOpportunities service =
```

Comment:
@force2b , the Opp trigger will evaluate Opps async now not just re-evaluate the Next Donation Date and Current/Next Year Values. Thus, if the Amount is manually changed on an *open* future Opp matching an installment, it will be reverted back in the Opp eval service since future open Opps matching installments are kept in sync with RD values.
FYI @sungysang